<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>vehiculeUI</class>
 <widget class="QDialog" name="vehiculeUI">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1039</width>
    <height>472</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QTableView" name="tableView">
   <property name="geometry">
    <rect>
     <x>210</x>
     <y>130</y>
     <width>811</width>
     <height>291</height>
    </rect>
   </property>
  </widget>
  <widget class="QWidget" name="layoutWidget">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>130</y>
     <width>171</width>
     <height>291</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <widget class="QLineEdit" name="nums">
      <property name="placeholderText">
       <string>nums</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLineEdit" name="numc">
      <property name="placeholderText">
       <string>numc</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QDateEdit" name="dateEdit"/>
    </item>
    <item>
     <widget class="QLineEdit" name="km">
      <property name="placeholderText">
       <string>km</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLineEdit" name="tel">
      <property name="placeholderText">
       <string>tel</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLineEdit" name="modele">
      <property name="placeholderText">
       <string>modele</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLineEdit" name="marque">
      <property name="placeholderText">
       <string>marque</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLineEdit" name="nom">
      <property name="placeholderText">
       <string>nom</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QWidget" name="layoutWidget">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>430</y>
     <width>195</width>
     <height>31</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout">
    <item>
     <widget class="QPushButton" name="ajouter">
      <property name="text">
       <string>Ajouter</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="editer">
      <property name="text">
       <string>Editer</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QWidget" name="layoutWidget">
   <property name="geometry">
    <rect>
     <x>770</x>
     <y>430</y>
     <width>241</width>
     <height>31</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout_2">
    <item>
     <widget class="QLineEdit" name="lineEdit_supp">
      <property name="placeholderText">
       <string>Supprimer</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="supp">
      <property name="text">
       <string>Supprimer</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>70</x>
     <y>20</y>
     <width>111</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>Recherche</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEdit_rech">
   <property name="geometry">
    <rect>
     <x>230</x>
     <y>50</y>
     <width>113</width>
     <height>24</height>
    </rect>
   </property>
  </widget>
  <widget class="QComboBox" name="comboBox_rech">
   <property name="geometry">
    <rect>
     <x>71</x>
     <y>50</y>
     <width>151</width>
     <height>24</height>
    </rect>
   </property>
   <item>
    <property name="text">
     <string>Numero de chassis</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>Numéro du véhicule</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>Model</string>
    </property>
   </item>
  </widget>
  <widget class="QPushButton" name="pushButton_rech">
   <property name="geometry">
    <rect>
     <x>350</x>
     <y>50</y>
     <width>80</width>
     <height>24</height>
    </rect>
   </property>
   <property name="text">
    <string>Recherche</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>480</x>
     <y>28</y>
     <width>70</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>Tri</string>
   </property>
  </widget>
  <widget class="QComboBox" name="comboBox_tri">
   <property name="geometry">
    <rect>
     <x>480</x>
     <y>50</y>
     <width>211</width>
     <height>24</height>
    </rect>
   </property>
   <item>
    <property name="text">
     <string>Kilométrage croissants</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>Kilométrage decroissants</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>Numéro du véhicule decroissants</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>Numéro du véhicule croissants</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>Date d'entrée decroissants</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>Date d'entrée croissants</string>
    </property>
   </item>
  </widget>
  <widget class="QPushButton" name="pushButton_stat">
   <property name="geometry">
    <rect>
     <x>250</x>
     <y>84</y>
     <width>131</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>Statistique</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_qrcode">
   <property name="geometry">
    <rect>
     <x>460</x>
     <y>84</y>
     <width>131</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>QrCode</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_3">
   <property name="geometry">
    <rect>
     <x>750</x>
     <y>30</y>
     <width>101</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>PDF</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEdit_pdf">
   <property name="geometry">
    <rect>
     <x>750</x>
     <y>50</y>
     <width>113</width>
     <height>24</height>
    </rect>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_pdf">
   <property name="geometry">
    <rect>
     <x>870</x>
     <y>50</y>
     <width>80</width>
     <height>24</height>
    </rect>
   </property>
   <property name="text">
    <string>Export</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton">
   <property name="geometry">
    <rect>
     <x>940</x>
     <y>100</y>
     <width>80</width>
     <height>24</height>
    </rect>
   </property>
   <property name="text">
    <string>Refresh</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="V_ID">
   <property name="geometry">
    <rect>
     <x>420</x>
     <y>430</y>
     <width>161</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QPushButton" name="Verif_button">
   <property name="geometry">
    <rect>
     <x>610</x>
     <y>430</y>
     <width>93</width>
     <height>28</height>
    </rect>
   </property>
   <property name="text">
    <string>Verifier</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_4">
   <property name="geometry">
    <rect>
     <x>290</x>
     <y>430</y>
     <width>101</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>IdV</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
